ci/ReleaseTool/ReleaseCommand.cs

```diff
@@ -222,9 +155,9 @@ public int Run()
                     // Merge into release
                     try
                     {
-                        mergeResult = gitHubClient.MergePullRequest(gitHubRepo, pullRequestId, PullRequestMergeMethod.Merge);
+                        mergeResult = gitHubClient.MergePullRequest(gitHubRepo, pullRequestId, PullRequestMergeMethod.Merge, $"Merging final GDK for Unreal {options.Version} release");
                     }
-                    catch (Octokit.PullRequestNotMergeableException e) {} // Will be covered by log below
```

Comment:
Feel like we should be grabbing the exception message from the exception no?

Reply:
Made it so